src/main/resources/db/migration/V0.2/V0.2.1__type5_uuids.sql

```diff
@@ -0,0 +1,25 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+-- update all UUIDs in database to be v5
+WITH _ AS (
+    UPDATE usage.interval_reading ir_outer SET block_uuid = uuid_generate_v5(
+        uuid_ns_url(),
+      (
+        SELECT ib.self_link_href FROM usage.interval_reading ir_inner
+          INNER JOIN usage.interval_block ib ON ir_inner.block_uuid = ib.uuid
+        WHERE ir_inner.id = ir_outer.id
+      )
+    )
+)
+UPDATE usage.interval_block SET uuid = uuid_generate_v5(uuid_ns_url(), self_link_href);
```

Comment:
We needed to be able to update multiple tables at once to avoid foreign key issues, so that's why this doesn't look like a single, simple `UPDATE` statement. I could've just done multiple `UPDATE`s in a transaction block, but I chose to use a CTE instead.